platform/linuxbsd/x11/display_server_x11.cpp

```diff
@@ -6261,8 +6261,14 @@ DisplayServerX11::DisplayServerX11(const String &p_rendering_driver, WindowMode
 
 #if defined(RD_ENABLED)
 	if (rendering_context) {
-		rendering_device = memnew(RenderingDevice);
```

Comment:
You removed one line too many here... not sure why CI didn't complain, but then it broke once merged: https://github.com/godotengine/godot/actions/runs/9723676850/job/26839084653

Reply:
Fixed up with 25de53e147a04ba15afc461b3ad4aa1884ff927d.

Reply:
> not sure why CI didn't complain

Seems like the CI never ran because GH was broken yesterday: https://www.githubstatus.com/incidents/9vwllhs2w1kj

I wish they'd make that more explicit before merging.